setup/cli/modules/deploy.php

```diff
@@ -151,11 +151,12 @@ function run($args, $options) {
 
         # Locate the upload folder
         $root = $this->find_root_folder();
+        $rootPattern = str_replace("\\","\\\\", $this->find_root_folder()); //need for windows case
```

Comment:
How about `$rootPattern = str_replace("\\","\\\\", $root);` so you don't need to call the find function again?